docs/core/docker/introduction.md

```diff
@@ -7,41 +7,62 @@ ms.custom: "mvc"
 
 # Introduction to .NET and Docker
 
-.NET can easily run in a Docker container. Containers provide a lightweight way to isolate your application from the rest of the host system, sharing just the kernel, and using resources given to your application. If you're unfamiliar with Docker, it's highly recommended that you read through Docker's [overview documentation](https://docs.docker.com/engine/docker-overview/).
+Containers are one of the most popular ways for deploying and hosting cloud applications, with tools like [Docker](https://www.docker.com/), [Kubernetes](https://kubernetes.io/), and [Podman](https://podman.io/). Many developers choose containers because it is straightforward to package an app with its required dependencies and then to get that app to reliably run on any container host. There is entensive support for [using .NET with containers](https://devblogs.microsoft.com/dotnet/category/containers/).
```

Comment:
```suggestion
Containers are one of the most popular ways for deploying and hosting cloud applications, with tools like [Docker](https://www.docker.com/), [Kubernetes](https://kubernetes.io/), and [Podman](https://podman.io/). Many developers choose containers because it's straightforward to package an app with its dependencies and get that app to reliably run on any container host. There's extensive support for [using .NET with containers](https://devblogs.microsoft.com/dotnet/category/containers/).
```